services/inbox/src/inbox/__init__.py

```diff
@@ -2,16 +2,16 @@
 from .mailgateway import get_server
 
 host = os.environ.get("EUCO_MAIL_SERVER_HOST", "")  # Defaults to localhost

```

Comment:
Not important, but I like the convention of having global scope variables in ALL CAPS to make the stand out as such. Not sure if that's a PEP8 thing or where it comes from.